CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### bugfix
+- Updated golang.org/x/crypto to v0.31.0
+
```

Comment:
this is not needed, [dependabot](https://github.com/newrelic/nri-kafka/pull/322) changes are taken into account automatically by the release toolkit